File: Back-End/MoviesServer/Movie.cs
using static System.Net.Mime.MediaTypeNames;
using System.IO;

namespace MoviesServer
{
    public class Movie
    {
        public int MovieID { get; set; }
        public string Title { get; set; }
        public string Director { get; set; }
        public int ReleaseYear { get; set; }
        public string Genre { get; set; }
        public double Rating { get; set; }
        public int Runtime { get; set; }
        public string Description { get; set; }
        public string Responses { get; set; }
        public string Image { get; set; }

        public Movie(int movieID, string title, string director, int releaseYear, string genre, double rating, int runtime, string description, string responses, string image)
        {
            MovieID = movieID;
            Title = title;
            Director = director;
            ReleaseYear = releaseYear;
            Genre = genre;
            Rating = rating;
            Runtime = runtime;
            Description = description;
            Responses = responses;
            Image = image;
        }
    }
}

File: Front-End/Entity/movie.py
class Movie:
    def __init__(self,movieID, title, director, release_year, genre, rating,
                 runtime, description, responses, image):
        self.movieID = movieID
        self.title = title
        self.director = director
        self.release_year = release_year
        self.genre = genre
        self.rating = rating
        self.runtime = runtime 
        self.description = description
        self.responses = responses
        self.image = image

    def __str__(self):
        return f"{self.title} ({self.release_year}), directed by {self.director}, Genre: {self.genre}, Rating: {self.rating}/10"
